yarn.lock

```diff
@@ -1578,22 +1578,6 @@
     enabled "2.0.x"
     kuler "^2.0.0"
 
-"@electron/get@^1.0.1":
```

Comment:
I've seen these changes in a few PRs, they relate the electron dependency being removed but the lock file not being updated. The first PR to go in should resolve these